uint8 action
geometry_msgs/PoseStamped ps
---
bool success